### PR TITLE
Add RSS 2.0 feed

### LINKED ISSUE #34: RSS Feed

### PR DESCRIPTION
**Pull request by Minoru**
Subject line sums it up nicely.

Don't hold anything back during review—it's my first time using Play! framework, I've got whooping 8 hours of experience with Scala, and I haven't seen MVC since the days of MFC assignments back in university.

Oh, and any help with that TODO item will be much appreciated.

Fixes #34

### CODE REVIEW

**Comment by rexim** [comment]
I'm taking this one

File: scalajvm/app/controllers/Quotes.scala [hunk 1]
```diff
@@ -21,6 +23,42 @@ object Quotes extends Controller {
     Ok(views.html.index(quotes, pageNumber, countPages, order, filter))
   }
 
+  def getRSSFeed() = ActionWithTx { request =>
+    import request.dbSession
+    import play.api.Play.current
+
+    val deploymentURL =
+      current.configuration.getString("application.deploymentUrl").get
```
Comment (rexim) [inline scalajvm/app/controllers/Quotes.scala]: There is no need for this deployment URL. You may want to take a look at how we generate the quote management URL in the email notifications: https://github.com/codingteam/loglist/blob/909960adfde5b2dd4c6713407019bfca6953b585/scalajvm/app/helpers/Notifications.scala#L35

File: scalajvm/app/controllers/Quotes.scala [hunk 2]
```diff
@@ -21,6 +23,42 @@ object Quotes extends Controller {
...
+        val cdataReady = description.replaceAll("]]>", "]]]]><![CDATA[>")
+        val escaped = scala.xml.Utility.escape(cdataReady)
+        val content = escaped.replaceAll("[\n\r]+", "<br/>")
+
+        new FeedItem(
+          quote.time,
+          new scala.xml.Atom("Quote #" + quote.id.toString()),
+          scala.xml.PCData(content),
+          // TODO: is there a better way to generate URLs?
+          deploymentURL + "quote/" + quote.id.toString(),
```
Comment (rexim) [inline scalajvm/app/controllers/Quotes.scala]: The previous comment is applicable to this line as well

**Comment by Minoru** [comment]
I've amended the commit according to @rexim's suggestions.

File: scalajvm/app/controllers/Quotes.scala [hunk 3]
```diff
@@ -21,6 +23,43 @@ object Quotes extends Controller {
...
+    import request.dbSession
+    import play.api.Play.current
+
+    val feedLimit = current.configuration.getInt("feed.limit").get
+
+    val quotes = QuoteQueries().getPageOfQuotes(0, feedLimit,
+        QuoteOrdering.Time, QuoteFilter.None)
+    val items = quotes.map(quote => {
+        val description = quote.content getOrElse ""
+        val cdataReady = description.replaceAll("]]>", "]]]]><![CDATA[>")
```
Comment (ForNeVeR) [inline scalajvm/app/controllers/Quotes.scala]: Could you please state a purpose of this thing?

**Comment by rexim** [comment]
LGBT

**Comment by rexim** [comment]
I'm merging that!

File: scalajvm/app/controllers/Quotes.scala (hunk 3 above)
Comment (Minoru) [inline scalajvm/app/controllers/Quotes.scala]: On line 45, I'm packing the description into CDATA. This line "escapes" characters that might break escaped version of description. See [Stack Overflow answer](http://stackoverflow.com/a/223782).

Comment (ForNeVeR) [inline scalajvm/app/controllers/Quotes.scala]: This sort of shit deserves at least a good bit of comment!

**Pull request merged**